main.nf

```diff
@@ -1,65 +1,96 @@
 #!/usr/bin/env nextflow 
 
-// Workflow to perform k-fold cross validation over a downloaded dataset (via accession id) or an input dataset
+// Workflow to perform k-fold cross validation over FTP downloaded datasets 
+// as specified in the csv file in 'data/datasets.txt' 
 
-// download query data
+// download data
 if(params.download_data.run == "True"){
-    query_data = params.download_data.query_output_dir
-    query_n_clust = params.download_data.query_num_clust.toString()
-    query_markers = "marker_genes_" + query_n_clust + ".tsv"
-
+	
+	// initialise required channels 
+	DATASET_IDS = Channel.create()
+	NUM_CLUST = Channel.create()
+	BARCODE_COLUMN = Channel.create()
+	CELL_LABEL_COLUMN = Channel.create()
```

Comment:
again, these channels can be removed, I presume

Reply:
True!